locations/locations.json

```
@@ -226,9 +280,14 @@
                 ],
                 "sections": [
                     {
-                        "name": "Cover Me With Diamonds",
+                        "name": "Have any type of diamond armor in your inventory.",
                         "access_rules": [
-                            "[$armor_diamond], @Overworld/Diamonds!/Diamonds!"
+                            "[$armor_diamond], @Overworld/Diamonds!/Consigue diamante"
```

wrong location name:
```suggestion
                            "[$armor_diamond], @Overworld/Diamonds!/Have a diamond in your inventory."
```